src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -96,7 +129,7 @@ const useGroupSetting = ({navigation, route}) => {
         }
       );
     } else {
-      SimpleToast.show(message, SimpleToast.SHORT);
+      Linking.openSettings();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Linking.openSettings()` function is now being used to open the app settings when the user doesn't grant the necessary permissions. This seems like a good change as it provides a better user experience by directly guiding the user to the settings page.

```diff
- SimpleToast.show(message, SimpleToast.SHORT);
+ Linking.openSettings();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->